### PR TITLE
Update vendor files, add Atk, rename gtksource into sourceview

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
cc @EPashkin @sdroege

### CODE REVIEW

File: gdk-pixbuf/vendor.md [hunk 1]
```diff
@@ -795,7 +782,7 @@ Queries a pointer to the pixel data of a pixbuf.
 # Returns
 
 A pointer to the pixbuf's pixel data.
-Please see the section on [image data](image-data) for information
+Please see the section on [image data][image-data] for information
```
Comment (EPashkin) [inline gdk-pixbuf/vendor.md:785]: Wrong link?

File: glib/vendor.md [hunk 2]
```diff
@@ -4693,10 +4725,10 @@ Determines if a given string is a valid D-Bus object path. You
 should ensure that a string is a valid D-Bus object path before
 passing it to `Variant::new_object_path`.
 
-A valid object path starts with '/' followed by zero or more
-sequences of characters separated by '/' characters. Each sequence
-must contain only the characters "[A-Z][a-z][0-9]_". No sequence
-(including the one following the final '/' character) may be empty.
+A valid object path starts with `/` followed by zero or more
+sequences of characters separated by `/` characters. Each sequence
```
Comment (EPashkin) [inline glib/vendor.md:4729]: Strange change, but it in GLib-2.0.gir.

File: gtk/vendor.md [hunk 3]
```diff
@@ -2539,7 +2490,7 @@ react to the text direction.
 
 [`ArrowExt`](trait.ArrowExt.html), [`MiscExt`](trait.MiscExt.html), [`WidgetExt`](trait.WidgetExt.html), [`glib::object::ObjectExt`](../glib/object/trait.ObjectExt.html), [`BuildableExt`](trait.BuildableExt.html), [`WidgetExtManual`](prelude/trait.WidgetExtManual.html), [`BuildableExtManual`](prelude/trait.BuildableExtManual.html)
 <!-- trait ArrowExt -->
-`[Deprecated since 3.14]` Trait containing all `Arrow` methods.
+Trait containing all `Arrow` methods.
```
Comment (EPashkin) [inline gtk/vendor.md:2493]: Seems cargo doc still add totally deprecated docs, this is wrong.

Comment (EPashkin) [reply gtk/vendor.md:2493]: * `gir doc`

**Comment by EPashkin** [comment]
👍 for merge as is, and fix errors in separate PR to minimize changes from it.

File: gdk-pixbuf/vendor.md (hunk 1 above)
Comment (GuillaumeGomez) [reply gdk-pixbuf/vendor.md:785]: I don't know actually...

File: gtk/vendor.md (hunk 3 above)
Comment (GuillaumeGomez) [reply gtk/vendor.md:2493]: Indeed, gir PR incoming I assume!

File: gdk-pixbuf/vendor.md (hunk 1 above)
Comment (EPashkin) [reply gdk-pixbuf/vendor.md:785]: Oh it in GdkPixbuf-2.0.gir too, then we can ignore it

**Comment by GuillaumeGomez** [comment]
Let's do it this way then. Thanks for your review!

**Pull request merged**